liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java

```diff
@@ -216,16 +216,14 @@ public void mpVersionTest() throws Exception {
       assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.0"), 1);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.3"), 2);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.4"), 3);
-      assertEquals(GenerateFeaturesMojo.getMPVersion("mpconfig-1.5"), 3);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.0"), 1);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.1"), 1);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.2"), 2);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.3"), 3);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.4"), 3);
-      assertEquals(GenerateFeaturesMojo.getMPVersion("mprestclient-1.5"), 3);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.0"), 1);
       assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.1"), 3);
-      assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.2"), 3);
+      assertEquals(GenerateFeaturesMojo.getMPVersion("mpjwt-1.2"), 4);
```

Comment:
Disabling these tests as they are failing with `21.0.0.9`.  `mpConfig-1.5` and `mprestclient-1.5` are not valid features. `mpjwt-1.2` is part of MP 4. 
@turkeylurkey please correct me if this is incorrect


See https://openliberty.io/docs/21.0.0.9/reference/feature/microProfile-3.3.html

Reply:
`mpConfig-1.5` and `mprestclient-1.5` were supposed to test handling of future releases of mp features but this is not a good testing approach as demonstrated by `mpjwt-1.2.` So yes please delete them.